test/exe/main_common_test.cc

```diff
@@ -200,6 +200,16 @@ TEST_P(AdminRequestTest, AdminRequestGetStatsAndKill) {
   EXPECT_TRUE(waitForEnvoyToExit());
 }
 
+// This test is the same as well, except we send ourselves a SIGINT, equivalent to receiving a
```

Comment:
Mind changing this to an absolute reference? ;-)
"This test is the same as AdminRequestGetStatsAndQuit, except....

Just in case folks aren't reading tests top-down.